software/apache/README.md

```diff
@@ -25,171 +21,144 @@ This guide walks you through setting up Apache natively on your MacOS with the a
+      fi
 
-    $ curl -L https://gist.githubusercontent.com/rhukster/f4c04f1bf59e0b74e335ee5d186a98e2/raw > /usr/local/bin/sphp
-    $ chmod +x /usr/local/bin/sphp
+      if [ -f "$conf_path" ]; then
+        echo "Updating listen port to $port_suffix in $conf_path"
+        sed -i.bak "s/listen = 127\.0\.0\.1\:90[[:digit:]][[:digit:]]/listen = 127.0.0.1:90$port_suffix/g" "$conf_path"
+      else
+        echo "Unable to locate fpm conf file for $version $conf_path"
+      fi
 
-Test PHP switcher by running
+      # Configure PHP-FPM to auto start using Homebrew services
+      brew services start "$formula"
+    done
 
-    $ sphp 7.0
+Important file locations:
 
-This should uncomment out the php 7.0 `LoadModule` line in `/usr/local/etc/httpd/httpd.conf` and make your shell's `php -v` report 7.0.*
-### Vhosts and the Hosts File
+    /usr/local/etc/php/5.6/php.ini
+    /usr/local/etc/php/7.0/php.ini
 
-Using Apache 2.4  you should be able to add vhosts using the `vhost` directory.
+## Setting up a new PHP site in Apache
 
-In your local apache `/usr/local/etc/httpd/` directory run the following commands.
+### Create the vHost file
+Each site will have it's own _vhost_ file in `/usr/local/etc/httpd/vhosts`.
 
-    $ mkdir vhosts
-    $ cd vhosts
-    $ touch <your_project_name>.conf
+:warning: You’ll need to swap out the following placeholders with your own settings:
 
-Open the file `<your_project_name>.conf`
+  * `<project_name>`: Domain safe name such as `seesparkbox`
+  * `<project_root>`: Web root such as `/projects/seesparkbox/static/dist`
+  * `<php_port>`: Port to the PHP-FPM instance, such as `9071` for 7.1
 
-Copy and paste the following vhost configuration inside that file:
+Copy, paste, & edit the following vhost configuration inside the
+file `/usr/local/etc/httpd/vhosts/<project_name>.conf`:
 
-    <VirtualHost 127.0.0.1:80>
-      ServerAdmin youremail@heysparkbox.com
-      DocumentRoot "<your_path_to_files>/<your_project_name>/web"
-      ServerName <your_project_name>.local
+    <VirtualHost *:80>
+        ServerName <project_name>.local
+        DocumentRoot "<project_root>"
+        ProxyPassMatch ^/(.*\.php(/.*)?)$ fcgi://127.0.0.1:<php_port><project_root>/$1
+        ErrorLog "/usr/local/var/log/httpd/<project_name>-error_log"
+        CustomLog "/usr/local/var/log/httpd/<project_name>-access_log" common
+        DirectoryIndex index.php index.html
 
-      <Directory "<your_path_to_files>/<your_project_name>/web">
-        Options Indexes MultiViews FollowSymLinks
-        AllowOverride All
-        Require all granted
-        DirectoryIndex index.php
-      </Directory>
+        <Directory "<project_root>">
+          AllowOverride All
+          DirectoryIndex index.php index.html
+          Options Indexes MultiViews FollowSymLinks
+          Require all granted
+       </Directory>
     </VirtualHost>
 
-You’ll need to swap out the bolded text with your own settings.
-
 Save this file, and restart apache.
```

Comment:
In addition to not recognizing `ProxyPassMatch`,  I also got a message saying that it could not determine _the server's fully qualified domain name _ and suggesting to set the `ServerName` globally.

Reply:
That's a common warning that can safely be ignored. Wouldn't hurt to mention it though.

Reply:
Same error here!